app/models/tslr_claim.rb

```diff
@@ -65,4 +71,16 @@ def ineligible_claim_school?
   def update_current_school
     self.current_school = employed_at_claim_school? ? claim_school : nil
   end
+
+  def normalise_trn
+    self.teacher_reference_number = normalised_trn
+  end
+
+  def normalised_trn
+    teacher_reference_number.gsub(/\D/, "")
+  end
+
+  def normalised_trn_length
+    errors.add(:teacher_reference_number, "Teacher reference number should contain seven digits") unless teacher_reference_number.present? && normalised_trn.length == TRN_LENGTH
```

Comment:
Small language tweak: I think the error message should use `must` instead of `should` as the length of the TRN is a hard requirement, not something that is suggestive or optional.